Fruitivy/Models/Produk.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Fruitivy.Models
+{
+    public class Produk
+    {
+        [Key]
+        public int ProdukId { get; set; }
+        public string Nama { get; set; }
+        public byte[] Picture { get; set; }
+        public int  Stok { get; set; }
```

Comment:
please remove space between int and Stok

Reply:
space is fixed